### PR TITLE
Configure s3 buckets for video images

### PR DESCRIPTION
**Pull request by mushtaqak**
This PR adds configuration for video image buckets.

https://openedx.atlassian.net/browse/EDUCATOR-42

edx-platform PR : https://github.com/edx/edx-platform/pull/15178
edx-val PR: https://github.com/edx/edx-val/pull/71

Configuration Pull Request
---

Make sure that the following steps are done before merging

  - [ ] A devops team member has commented with :+1:
  - [ ] are you adding any new default values that need to be overridden when this goes live?
    - [ ] Open a ticket (DEVOPS) to make sure that they have been added to secure vars.
    - [ ] Add an entry to the CHANGELOG.

### CODE REVIEW

**Comment by mushtaqak** [comment]
@feanil @jibsheet Can you review? What do I need to get this PR merged?

**Comment by jibsheet** [comment]
It doesn't merge until it's planned and applied.
Feanil is making some changes to support you adding a test env in Sandbox, so I suspect we'll make that part of this PR.

**Comment by jibsheet** [comment]
@mushtaqak to aid review can you link to your platform changes, so we can see the aws.py wiring?

**Comment by mushtaqak** [comment]
@jibsheet Added edx-platform PR and edx-val PR links. Thanks

**Comment by mushtaqak** [comment]
@jibsheet This is also ready, can you review this too?

**Comment by jibsheet** [comment]
@mushtaqak can you rebase to resolve the CHANGELOG.md conflict and also add a comment above EDXAPP_VIDEO_IMAGE_SETTINGS saying something like this is a django storages config, you can configure S3 or Swift here (You may just be pointing to docs you already wrote over in common.py in the platform.  Trying to help our openedx deployers here).

**Comment by mushtaqak** [comment]
@jibsheet Please review.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -665,6 +665,14 @@ EDXAPP_SESSION_SAVE_EVERY_REQUEST: false
 
 EDXAPP_SESSION_COOKIE_SECURE: false
 
+EDXAPP_VIDEO_IMAGE_MAX_AGE: 31536000
```
Comment (jibsheet) [inline playbooks/roles/edxapp/defaults/main.yml:667]: If you want this to end up in lms.env.json, you need to refer to it below (similar to what happens with VIDEO_IMAGE_SETTINGS)

Comment (mushtaqak) [reply playbooks/roles/edxapp/defaults/main.yml:667]: Addressed feedback

**Comment by mushtaqak** [comment]
@jibsheet feature branch is ready to be merge https://github.com/edx/edx-platform/issues/15178

**Review by jibsheet — APPROVED** [review]

**Pull request merged**